README.md

```diff
@@ -4,10 +4,45 @@
 
 ## Summary
 
-The objective of this package is to check the health of different front-end projects and to see if they are using good practices.
+The objective of this package is to analyze, check and inform the health of different front-end projects regarding the use of good practices on the code, SEO, project and repository configurations depending on the technology being used on it.
+
+---
 
 ## Usage
 
-TBD
+Once positioned on the root directory of the project run the following command:
+
+>`node index.js`
+
+
+#### Parameters
+
+|<div style="width:250px">Parameter</div>|Description|If omitted|
+|---|---|---|
+|`-p [relativePath/to/project]`|Indicates the path to the project to check|The checks will be run against everything inside */test* directory.|
+| `-t [angular/react/vue]`|Choose which technology checks applies to your project|Will use react as default technology|
+|`-r [repository name]`|Specify the repository name on Github||
+|`-o [organization name]`|Specify the organization which owns the repository||
+|`--onlyGit`|If this argument is passed it will only run the checks regarding the Github repository|Will run all the checks|
```

Comment:
To run the gitChecks you need to have a .env variable called `OAUTH_TOKEN` with your github token credentials. If not you won't be able to run the checks duw to credentials problems